Tests/Brokerages/Alpaca/AlpacaBrokerageTests.cs

```diff
@@ -0,0 +1,268 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Concurrent;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using QuantConnect.Brokerages.Alpaca;
+using QuantConnect.Configuration;
+using QuantConnect.Interfaces;
+using QuantConnect.Orders;
+using QuantConnect.Securities;
+
+namespace QuantConnect.Tests.Brokerages.Alpaca
+{
+    [TestFixture, Ignore("This test requires a configured and testable Alpaca practice account")]
+    public partial class AlpacaBrokerageTests : BrokerageTests
+    {
+        /// <summary>
+        ///     Creates the brokerage under test and connects it
+        /// </summary>
+        /// <returns>A connected brokerage instance</returns>
+        protected override IBrokerage CreateBrokerage(IOrderProvider orderProvider, ISecurityProvider securityProvider)
```

Comment:
I think it would be great to have some extra tests covering the `Brokerage` API, for example `CancelOrder`, `GetCashBalance`, `GetOpenOrders`, `GetAccountHoldings`

Reply:
These extra tests are covered in the `BrokerageTests` base class